metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -388,8 +424,13 @@ def emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
                     title=dashboard_data.get("dashboard_title", ""),
                     entity_urn=dashboard_snapshot.urn,
                 )
+            except Exception as e:
+                logger.warning(
+                    f"Failed to emit dashboard MCE {dashboard_data.get('dashboard_title')}. Error: \n{e}"
```

Comment:
this would be an error in fetching/extraction, not emission - right?

Reply:
Hm I guess it would include both since the emission is also in the try block. I can look to separate the two.

Reply:
this would actually be just fetching/extraction.

after the `yield ...` statement, the datahub sink is responsible for emitting the entity and will catch exceptions there. so errors during the emit will not show up in the source